packages/cli/package.json

```diff
@@ -21,6 +21,9 @@
     "node": "^20.0.0 || ^22.0.0"
   },
   "dependencies": {
+    "@commander-js/extra-typings": "^12.1.0",
+    "adm-zip": "^0.5.16",
+    "archiver": "^5.3.2",
```

Comment:
Seems like we are using two zip libraries. Is there any way we can get away with just one?